examples/tls-wolfssl-posix/wolfssl-server.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2019 Kaleb J. Himes <kaleb@wolfssl.com>
+ *
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       wolfSSL server example
+ *
+ * @author      Kaleb J. Himes <kaleb@wolfssl.com>
+ *
+ * @}
+ */
+
```

Comment:
please add the following here so that strnlen is provided on all platforms:
```c
/* Required for strnlen in string.h, when building with -std=c99 */
#define _DEFAULT_SOURCE 1 
```